### PR TITLE
KTL-840; update dependencies for last UI library usage

### PR DESCRIPTION
**Pull request by zoobestik**
Demo: https://zoobestik.github.io/kotlinfoundation.github.io/

### CODE REVIEW

File: .github/workflows/gh-pages-deploy.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v3
         with:
-          node-version: '16'
+          node-version: '18'
```
Comment (zoobestik) [inline .github/workflows/gh-pages-deploy.yml:19]: Required 18+ for gatsby@^5

File: gatsby-config.js [hunk 2]
```diff
@@ -13,16 +13,13 @@ module.exports = {
...
       options: {
-        icon: "node_modules/@jetbrains/kotlin-web-site-ui/dist/svg/kotlin_64.svg",
+        icon: "node_modules/@jetbrains/kotlin-web-site-ui/out/svg/kotlin_64.svg",
       },
     },
     {
       resolve: "gatsby-plugin-mdx",
       options: {
         extensions: [ '.md' ],
-        defaultLayouts: {
```
Comment (zoobestik) [inline gatsby-config.js:23]: Function was [dropped](https://www.gatsbyjs.com/plugins/gatsby-plugin-mdx/#layouts) :(

File: package.json [hunk 3]
```diff
@@ -8,44 +8,56 @@
   ],
   "scripts": {
     "start": "gatsby develop --port 8082",
-    "build": "gatsby build",
+    "build": "gatsby build --prefix-paths",
```
Comment (zoobestik) [inline package.json:11]: Just for deploy to demo

File: src/pages/guidelines.md [hunk 4]
```diff
@@ -42,7 +44,7 @@ Please follow these guidelines when referencing the Trademark:
 example "Kotlin™ ..." This requirement is waived in all contexts where such marks are not normally included, such as non-marketing emails, online discussions, and academic papers. We encourage the use of the TM symbol whenever possible, but recognize that many non-commercial and informal uses will omit it.
 * Where identifying that a product or service is built on the Kotlin programming language or runs the Kotlin 
 programming language, use the product’s own name followed by “in Kotlin™”, “for Kotlin™”, “compatible with Kotlin™”, “running Kotlin™” etc.  Do not incorporate Kotlin into the product name. 
-* User groups, communities and/or meet-ups should follow the naming convention "{XYZ} Kotlin User Group" where {XYZ} 
+* User groups, communities and/or meet-ups should follow the naming convention "\{XYZ\} Kotlin User Group" where \{XYZ\}
```
Comment (zoobestik) [inline src/pages/guidelines.md:47]: Currently, `{XYZ}` is insert global variable usage :(

File: src/pages/index.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ layout: "wide"
 whiteHeader: true
 contactUs: true
 ---
-
+export {Layout as default} from '../components/Layout';
```
Comment (zoobestik) [inline src/pages/index.md:7]: See comment:
> Function was [dropped](https://www.gatsbyjs.com/plugins/gatsby-plugin-mdx/#layouts) :(

File: gatsby-config.js (hunk 2 above)
Comment (nikpachoo) [reply gatsby-config.js:23]: Ooh, okey :( 
Could you please add a remark to the "How to Add a Page" section in the Readme.md about adding the layout import?

**Review by nikpachoo — APPROVED** [review]

**Pull request merged**